<ui version="4.0" >
 <class>PathEditorBase</class>
 <widget class="QWidget" name="PathEditorBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Path Editor</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <widget class="QComboBox" name="mPathCB" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Select or type a path to see the matches below.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="styleSheet" >
      <string/>
     </property>
     <property name="editable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="mPathLbl" >
     <property name="text" >
      <string>Path:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Matches:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QListWidget" name="mMatchesList" >
     <property name="toolTip" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">List of matches based on the path given above.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
